themes/scss/map/dashboard-info.scss

```diff
@@ -93,9 +91,9 @@
 }
 .CDB-Dashboard-infoDescription {
   color: #FFF;
-  font-size: $sFontSize-normal;
+  font-size: $sFontSize-medium;
   font-weight: $sFontWeight-lighter;
-  line-height: $sLineHeight-normal;
+  line-height: $sLineHeight-medium;
```

Comment:
Name of variable `sLineHeight-medium` should be written in all lowercase letters with hyphens instead of underscores